test/Numerics/Mesh/DSS.jl

```diff
@@ -0,0 +1,166 @@
+using Test
```

Comment:
Might be nice to have a test that does more than just count the number of vertices.

Even a small 2 element mesh, initialize with some known numbers and make sure we get out what we expect.